deploy/helm/values.yaml

```diff
@@ -754,3 +755,8 @@ external-dns:
     operator: Equal
     value: "true"
     effect: NoSchedule
+
+datascriptClients:
+  registry: registry.cn-hangzhou.aliyuncs.com
+  repository: apecloud/datascript-clients
+  tag: 0.1.0
```

Comment:
this tag should be consistent with the image that build by the CD workflow, or use empty string

Reply:
Updated. Pls check.

Reply:
Updated. Pls check.